src/nbmake/nb_run.py

```diff
@@ -1,19 +1,34 @@
 from pathlib import Path
-from typing import Any, Dict, Optional
+from typing import Any, Dict, List, Optional
 
 import nbformat
 from nbclient.client import (
     CellExecutionError,
     CellTimeoutError,
     NotebookClient,
 )
+from nbclient.util import run_sync
 from nbformat import NotebookNode
+import jupytext
```

Comment:
Could we (a) add this to project dependencies if we think it's reliable and (b) Only import it when used on line 27 if possible